lib/bypass.ex

```diff
@@ -223,6 +223,21 @@ defmodule Bypass do
   def expect_once(%Bypass{pid: pid}, method, path, fun),
     do: Bypass.Instance.call(pid, {:expect_once, method, path, fun})
 
+  @doc """
+  Allows the function to be invoked zero or many times regardless of the route
+  or HTTP method.
+
+  ```elixir
+  Bypass.stub(bypass, fn conn ->
+    Agent.get_and_update(AgentModule, fn step_no -> {step_no, step_no + 1} end)
+    Plug.Conn.resp(conn, 429, ~s<{"errors": [{"code": 88, "message": "Rate limit exceeded"}]}>)
+  end)
+  ```
+  """
+  @spec stub(Bypass.t(), (Plug.Conn.t() -> Plug.Conn.t())) :: :ok
+  def stub(%Bypass{pid: pid}, fun),
+    do: Bypass.Instance.call(pid, {:stub, :any, :any, fun})
+
```

Comment:
Instead of creating new function, please just update documentation and specs for `stub/4` function - `method` and `path` can be `String.t() | :any`. Creating a stub for all paths and methods should be implicit, what is not clear when calling `stub/2`. 

Please also add some examples how it can be used.